students/HarveyHerela/lesson07/html_render_test.py

```diff
@@ -0,0 +1,99 @@
+from io import StringIO
+import unittest
+
+import html_render
+
+class HtmlRenderTest(unittest.TestCase):
+
+    def test_basic_tags(self):
+        tags = ('Body', 'Head', 'Ul', 'Li')
+        for t in tags:
+            expected = '<{tag}>\n\n</{tag}>'.format(tag=t.lower())
+            page = eval('html_render.{tag}()'.format(tag=t))
```

Comment:
Not a concern here obviously, but you generally want to avoid using `eval` in production code for reasons detailed in this post by a prominent Python dev/thought leader: 

https://nedbatchelder.com/blog/201206/eval_really_is_dangerous.html